### PR TITLE
docs(WEBRTC-266-267): add typedoc on local and remote stream properties

### PR DESCRIPTION
**Pull request by DeividVeloso**
-  add typedoc on local and remote stream properties

## 📝 To Do

- [x] All linters pass
- [x] All tests pass
- [ ] Change documentation based on my changes

## ✋ Manual testing

1. Internal documentation

### CODE REVIEW

File: packages/js/src/Modules/Verto/webrtc/BaseCall.ts [hunk 1]
```diff
@@ -138,10 +138,32 @@ export default abstract class BaseCall implements IWebRTCCall {
...
+  /**
+   * Gets the local stream of the call.
+   * This can be used in a video/audio element to play the local media.
+   * [MediaStream](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream).
+   *
+   * @examples
+   *
+   * ```js
+   * call.localStream();
+   * ```
```
Comment (hugobdev) [inline packages/js/src/Modules/Verto/webrtc/BaseCall.ts:150]: Let's add an example of how to get this into a `video` or `audio` element with `srcObject`.

File: packages/js/src/Modules/Verto/webrtc/BaseCall.ts [hunk 2]
```diff
@@ -138,10 +138,32 @@ export default abstract class BaseCall implements IWebRTCCall {
...
+  /**
+   * Gets the remote stream of the call.
+   * This can be used in a video/audio element to play the local media.
+   * [MediaStream](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream).
+   *
+   * @examples
+   *
+   * ```js
+   * call.remoteStream();
+   * ```
```
Comment (hugobdev) [inline packages/js/src/Modules/Verto/webrtc/BaseCall.ts:165]: Let's add an example of how to get this into a `video` or `audio` element with `srcObject`.

**Review by hugobdev — CHANGES_REQUESTED** [review]

File: packages/js/src/Modules/Verto/webrtc/BaseCall.ts (hunk 2 above)
Comment (DeividVeloso) [reply packages/js/src/Modules/Verto/webrtc/BaseCall.ts]: Done!

File: packages/js/src/Modules/Verto/webrtc/BaseCall.ts (hunk 1 above)
Comment (DeividVeloso) [reply packages/js/src/Modules/Verto/webrtc/BaseCall.ts]: Done!

File: packages/js/src/Modules/Verto/webrtc/BaseCall.ts [hunk 3]
```diff
@@ -138,10 +138,34 @@ export default abstract class BaseCall implements IWebRTCCall {
...
+  /**
+   * Gets the local stream of the call.
+   * This can be used in a video/audio element to play the local media.
+   * [MediaStream](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream).
+   *
+   * @examples
+   *
+   * ```js
+   * const stream = call.localStream();
+   * <video id="localId" srcObject={stream} autoPlay='autoplay' controls={false} />
```
Comment (hugobdev) [inline packages/js/src/Modules/Verto/webrtc/BaseCall.ts:150]: Actually, this is not valid JS. You could do it like:
```suggestion
   * document.querySelector('video').srcObject = stream;
```

File: packages/js/src/Modules/Verto/webrtc/BaseCall.ts [hunk 4]
```diff
@@ -138,10 +138,34 @@ export default abstract class BaseCall implements IWebRTCCall {
...
+  /**
+   * Gets the remote stream of the call.
+   * This can be used in a video/audio element to play the local media.
+   * [MediaStream](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream).
+   *
+   * @examples
+   *
+   * ```js
+   * const stream = call.remoteStream();
+   * <video id="remoteId" srcObject={stream} autoPlay='autoplay' controls={false} />
```
Comment (hugobdev) [inline packages/js/src/Modules/Verto/webrtc/BaseCall.ts:166]:
```suggestion
   * document.querySelector('video').srcObject = stream;
```

**Review by hugobdev — CHANGES_REQUESTED** [review]

File: packages/js/src/Modules/Verto/webrtc/BaseCall.ts [hunk 5]
```diff
@@ -138,10 +138,34 @@ export default abstract class BaseCall implements IWebRTCCall {
...
+  /**
+   * Gets the local stream of the call.
+   * This can be used in a video/audio element to play the local media.
+   * [MediaStream](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream).
+   *
+   * @examples
+   *
+   * ```js
+   * const stream = call.localStream();
+   * document.querySelector('video').srcObject = stream;
```
Comment (hugobdev) [inline packages/js/src/Modules/Verto/webrtc/BaseCall.ts:150]:
```suggestion
   * document.querySelector('audio').srcObject = stream;
```

File: packages/js/src/Modules/Verto/webrtc/BaseCall.ts [hunk 6]
```diff
@@ -138,10 +138,34 @@ export default abstract class BaseCall implements IWebRTCCall {
...
+  /**
+   * Gets the remote stream of the call.
+   * This can be used in a video/audio element to play the local media.
+   * [MediaStream](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream).
+   *
+   * @examples
+   *
+   * ```js
+   * const stream = call.remoteStream();
+   * document.querySelector('video').srcObject = stream;
```
Comment (hugobdev) [inline packages/js/src/Modules/Verto/webrtc/BaseCall.ts:166]:
```suggestion
   * document.querySelector('audio').srcObject = stream;
```

I think audio is the safest choice here as it should be the most common use case right now

**Review by hugobdev — APPROVED** [review]

**Pull request merged**